src/ui/menu.c

```diff
@@ -132,14 +132,14 @@ enum {
 
 static bool settings_nav_callback(uint8_t page, nbgl_pageContent_t* content) {
     if (page == 0) {
-        content->type = SWITCHES_LIST;
-        content->switchesList.nbSwitches = NB_SETTINGS_SWITCHES;
-        content->switchesList.switches = (nbgl_layoutSwitch_t*) G_switches;
-    } else if (page == 1) {
         content->type = INFOS_LIST;
         content->infosList.nbInfos = ARRAY_COUNT(info_types);
         content->infosList.infoTypes = (const char**) info_types;
```

Comment:
cast not needed anymore, same just below.

Reply:
Done